tests/client/core/reducers/test_categories.js

```diff
@@ -126,6 +133,18 @@ describe('categories reducer', () => {
               slug: 'anime',
               type: ADDON_TYPE_THEME,
             },
+            naturé: {
```

Comment:
since this entire `ADDON_TYPE_THEME` object is a duplication of the Firefox one, how about pulling it out into a shared constant? I think that will also make the intention of the test case more clear.